src/kirby/components/modal/services/modal.controller.ts

```diff
@@ -7,11 +7,20 @@ import { ActionSheetHelper } from './action-sheet.helper';
 import { ModalConfig } from '../modal-wrapper/config/modal-config';
 import { ActionSheetConfig } from '../action-sheet/config/action-sheet-config';
 import { AlertConfig } from '../alert/config/alert-config';
+import { KirbyAnimation } from '@kirbydesign/designsystem/animation/kirby-animation';
+
+export const modalScrollNoop = () => {
+  throw new Error('No modal windows are currently registered');
+};
+
+export type modal = { close: (data?: any) => {}; config?: ModalConfig };
```

Comment:
Wouldn't it be more appropriate to use an interface for this?

Regardless of using `type` or `interface`, they are (by convention) always written with an uppercase first letter.

Reply:
Yes, let's do that